tests/test_scene_rendering/test_caching_relalted.py

```diff
@@ -1,53 +0,0 @@
-import os
```

Comment:
**Question** Why this file get removed ? 
I guess it's temporary, I just want to be sure it's intentional :D

Reply:
There's a new file with the name spelled properly.

Reply:
It's `test_caching_related.py`, you can see it if you browse the code from the branch being merged.